README.md

```diff
@@ -66,21 +80,25 @@ yarn start
 
 Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the contract component or the example ui in the frontend. You can tweak the app config in `packages/nextjs/scaffold.config.ts`.
 
-Run smart contract test with `yarn hardhat:test`
+Run smart contract test with `yarn hardhat:test` or `yarn foundry:test` depending of your solidity framework.
 
-- Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
+- Edit your smart contract:
+  - Hardhat => `YourContract.sol` in `packages/hardhat/contracts`
+  - Foundry => `YourContract.sol` in `packages/foundry/src`
```

Comment:
same https://github.com/scaffold-eth/scaffold-eth-2/pull/442/files#r1281921252

Reply:
Updated in both places 👌